JsonClient/JsonClient/Program.cs

```diff
@@ -70,9 +88,9 @@ static async Task Main(string[] args)
 
                         break;
                     case 3:
-                       //comment = await client.PostAsync( GetComment());
-                       if(comment != null){
-                       //Console.WriteLine("name: {0}\nemail: {1}\nbody: {2}", comment.name, comment.email, comment.body);
+                       result = await client.PostAsync( GetObject());

```

Comment:
have a problem here because I need to put a parameter Also if I put it will have another issue because the method return an object and result is TObject @danielportilloh

Reply:
fixed